src/System.Drawing.Common/ref/System.Drawing.Common.csproj

```diff
@@ -18,4 +18,15 @@
     <ProjectReference Include="..\..\System.Runtime.Extensions\ref\System.Runtime.Extensions.csproj" />
     <ProjectReference Include="..\..\System.Runtime.InteropServices\ref\System.Runtime.InteropServices.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp3.0'">
```

Comment:
nit: Use TargetFramework instead of TargetGroup for version specific checks.

Reply:
@ViktorHofer why did we change this? We've always used TargetGroup in the past, did we change the guidelines?

Reply:
When we enabled project restore for test projects we had to change a bunch of places to allow restoring the right assets when multiplexing on TargetFrameworks. As we will enable project restore for the rest of the projects as well in the near future the guideline changed slightly:

1. If possible use TargetsX, i.e. TargetsNetCoreApp, TargetsNetFx, TargetsUap (or TargetFrameworkIdentifier)
2. If you need to condition on a specific tfm, use TargetFramework
3. If you need to condition on the custom, version-less tfm, use TargetGroup.